nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -177,10 +177,13 @@
   data-files:
     - dirs:
         - 'resources/system'
+    - patterns:
+        - 'VERSION'
   anti-bloat:
     - no-auto-follow:
-        'examples': 'Arcade examples are not be available unless you do "--include-module=acade.examples"'
+        'examples': 'Arcade examples are not available unless you do "--include-module=acade.examples"'
         'experimental': 'Experimental features of Arcade will not be available unless you do "--include-module=acade.experimental"'
```

Comment:
Drive-by review: the suggestion text has the package's name misspelled (`acade` instead of `arcade`).

Reply:
Thank you for the heads up!